src/styles/variables.css

```diff
@@ -14,4 +15,10 @@
 	--title-bar-color: var(--dojo-white);
 	--underlay-bg: var(--dojo-shade-gray);
 	--slide-panel-width: 256px;
+
+	--button-bg: var(--dojo-blue);
```

Comment:
Is there another name you'd prefer? I was going by the `specific usage name = color name` pattern, but I'm open to other naming patterns. I'm not sure what a more generic variable would be in this case, though. It wouldn't necessarily be the same as a link; what else is there?